fastapi/exceptions.py

```diff
@@ -1,20 +1,8 @@
-from typing import Any, Dict, Optional, Sequence, Type
+from typing import Any, Sequence, Type
 
 from pydantic import BaseModel, ValidationError, create_model
 from pydantic.error_wrappers import ErrorList
-from starlette.exceptions import HTTPException as StarletteHTTPException
-
-
-class HTTPException(StarletteHTTPException):
-    def __init__(
-        self,
-        status_code: int,
-        detail: Any = None,
```

Comment:
There are several examples and places in the code base that expect `detail` to be any JSON-able data, not only a string (as is with Starlette).

So, only for this little type hint we still need the custom `HTTPException` class in FastAPI.